docs/en/reference/configuration.rst

```diff
@@ -243,18 +243,23 @@ Setting ``transactional`` to ``false`` will disable that.
 From the Command Line
 ~~~~~~~~~~~~~~~~~~~~~
 
-You can also set this option from the command line with the ``migrate`` command and the ``--all-or-nothing`` option:
+To enable all the migrations to complete together, you can also set this option from the command line with
```

Comment:
I'm not sure people will understand what is meant by "completing together". I think we might not want to shy away from being technical here, talking about transactions should make clarify things IMO.

Reply:
Pushed a new phrasing, let me know if it's ok the way is written now.